packages/itwin/tree-widget/src/components/search-bar/SearchBar.scss

```diff
@@ -79,12 +67,12 @@
       &.show {
         visibility: visible;
         opacity: 1;
-        width: 100%;
+        width: calc(100% + 32px);
```

Comment:
Why do you need to add `32px`?

Reply:
The main idea was to overflow the dropdown button, but now the dropdown button won't be rendered if enableGrouping is set to false